projects/components/src/stepper/stepper.component.scss

```diff
@@ -0,0 +1,56 @@
+@import 'mixins';
+
+:host {
+  ::ng-deep {
+    .mat-step-header {
+      .mat-step-label {
+        @include overline($gray-5);
+        text-transform: uppercase;
+      }
+
+      .mat-step-label-selected {
+        @include overline($gray-7);
+        text-transform: uppercase;
+      }
+
+      .mat-step-icon {
+        @include overline(white);
+        @include frame-border;
+        border-color: $blue-4;
+        color: $blue-4;
+      }
+
+      .mat-step-icon-selected {
+        background-color: $blue-4;
+        color: white;
+      }
+    }
+
+    .mat-vertical-stepper-header {
+      padding: 24px;
+    }
+
+    .mat-vertical-content-container.mat-stepper-vertical-line::before {
+      top: -16px;
+      bottom: -16px;
+      border-left-color: $gray-9;
+    }
+  }
+}
+
+.stepper {
+  @include flex-layout(column);
+  justify-content: space-between;
+
+  .action-buttons {
+    display: flex;
+    justify-self: flex-end;
```

Comment:
Could you use here an align-self property to do the same?

Reply:
Oh, we don't require both anymore. It's working fine with flex.

Reply:
Removed.